FinniversKit/Sources/Fullscreen/TransactionView/TransactionStepView/TransactionStepViewModel.swift

```diff
@@ -6,12 +6,20 @@ import Foundation
 
 public protocol TransactionStepViewModel {
     var state: TransactionStepViewState { get }
-    var customBackground: TransactionStepViewCustomBackground? { get }
-    var title: String { get }
+    var style: TransactionStepView.CustomStyle? { get }
+    var main: TransactionStepContentViewModel? { get }
+    var detail: TransactionStepContentViewModel? { get }
+}
+
+public protocol TransactionStepContentViewModel {
+    var title: String? { get }
     var body: NSAttributedString? { get }
+    /*
+     For certain steps the attributed string assigned to body will contain a href element.
+     The host app will remove the <a href> element and the nativeButton will be rendered above the primaryButton.
```

Comment:
Does this mean the app will need to parse the attributed string to remove any link elements? Or will the backend remove it?

Reply:
Yes, unfortunately the link element will be removed client side. I would like do this on the backend, but due to democracy my suggestion was not favoured 😞

Reply:
This is now redundant and will be fixed in the next PR.